zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -545,6 +546,7 @@ def flow(  # pylint:disable=too-many-branches, too-many-statements
             logger.info(f"Submitting {len(tasks)} processing tasks from operation {self.op}.")
             yield tasks
         elif not self.use_checkerboarding and self.force_intermediaries:
+            assert dst is not None  # no intermediaries without destination layer
```

Comment:
Also unnecessary.

Reply:
I suspect that when we (or some new guy) come back to this code 3 years from now, we will no longer remember why it is safe to assert that dst is not None here (or in the previous case).  But I trust your judgement; out it goes.

Reply:
That's a good point, but I think this needs to actually be caught in `subchunkable_apply_flow.py` - if `use_checkerboarding` (which is inferred) or `force_intermediaries` (given directly) is set when `dst` is None, that should raise an exception before it gets to this stage.

Reply:
(Theoretically, the user may use `VolumetricApplyFlow` as a standalone, but that's not really expected - we used to have a `build_` method in there but removed it with the expectation that users will use subchunkable instead.)